doc/tutorial/02_index/index.md

```diff
@@ -1,19 +1,274 @@
+`--shape`.
+
+\note
+Raptor also has an advanced help page for the experimental content, which can be accessed as usual by typing
+`raptor build -hh` or `raptor build --advanced-help`.
+
+## Others
+
+### Compressed index
+
+The raptor index can also be stored in a compressed form (`--compressed`). This is only worthwhile if you want a very
+low false positive rate, because then you have many bits equal 0 that can be compressed well.
+
+### Parallelization
+
+Raptor supports parallelization. By specifying `--threads`, for example, the fastq-records are processed simultaneously.
+
+### Partitioned indices
+
+To reduce the overall memory consumption of the search, the index can be divided into multiple (a power of two) parts.
+This can be done by passing `--parts n`, where n is the number of parts you want to create. This will create `n` files,
+each representing one part of the index.
+The `--size` parameter describes the overall size of the index. For example, `--size 8g --parts 4` will create four
+2 GiB indices. This will reduce the memory consumption of `raptor build` and `raptor search` by approximately 6 GiB,
+since there will only be one part in memory at any given time. `raptor search` will automatically detect the parts, and
+does not need any special parameters.
+
+## IBF vs HIBF {#hibf}
+
+/todo This is a placeholder section and needs more information including the chopper layout.
+
+Raptor works with the Interleaved Bloom Filter by default. A new feature is the Hierarchical Interleaved Bloom Filter
+(HIBF) (raptor::hierarchical_interleaved_bloom_filter), which can be used with `--hibf` can be used. This uses a more
+space-saving method of storing the bins. It distinguishes between the user bins, which reflect the individual samples as
+before, and the so-called technical bins, which throw some bins together. This is especially useful when there are
+samples of very different sizes.
+Since the HIBF calculates the size of the index itself, it is no longer possible to specify a size here. But we can
+offer the option to name the desired false positive rate with `--fpr`.
+
+```bash
+raptor build --kmer 19 --hash 3 --hibf --fpr 0.1 --output raptor.index all_bin_paths.txt
+```
+
+\note
+For a detailed explanation of the Hierarchical Interleaved Bloom Filter (HIBF), please refer to the
+`raptor::hierarchical_interleaved_bloom_filter` API.
+
+\assignment{Assignment 4: HIBF}
+Lets use the HIBF for our small example. Thus run the example above with a false positive rate of 0.05 and call the new
+index `hibf_raptor.index`.
+As our example is small, we will keep the kmer size of 4
+\hint
+...
+\endhint
+\endassignment
+
+\solution
+You should have run:
+```bash
+raptor build --kmer 4 --hibf --fpr 0.05 --output hibf_raptor.index all_paths.txt
+```
+/todo Currently not working: `[Error] The list of input files cannot be empty.`
+
+Your directory should look like this:
+```bash
+tmp$ ls -la
+...
```

Comment:
... ? :)

Reply:
Since calling `raptor build` with hibf does not work yet (chopper is missing), I do not have any files for the `ls` command. Therefore, only the ls command is listed here as a placeholder. I would then adapt this in the sense of layout building.
The whole section is still a placeholder.